src/condor_io/condor_auth_ssl.cpp

```diff
@@ -1618,6 +1618,22 @@ long Condor_Auth_SSL :: post_connection_check(SSL *ssl, int role )
 		}
 	}
 
+	if (mySock_->isClient()) {
+		std::unique_ptr<BIO, decltype(&BIO_free)> bio(BIO_new( BIO_s_mem() ), BIO_free);
+
+		if (!PEM_write_bio_X509(bio.get(), cert)) {
+			dprintf(D_SECURITY, "Unable to convert server host cert to PEM format.\n");
+			goto err_occured;
+		}
+		char *pem_raw;
+		auto len = BIO_get_mem_data(bio.get(), &pem_raw);
+		if (len) {
+			classad::ClassAd ad;
+			ad.InsertAttr("ServerPublicCert", pem_raw, len);
```

Comment:
The attribute name should be added to condor_attributes.h and the macro used here.